CHANGELOG

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Improvements
 
 - Added basic access authentication (all methods except /info)
+- tabpy-tools can deploy to TabPy with auth on
```

Comment:
should this be

tabpy-tools can deploy models to TabPy with authentication on

Reply:
fixed